pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [build-system]
-requires = ["setuptools", "wheel"]
-build-backend = "setuptools.build_meta"
+requires = ["uv_build >= 0.7.19, <0.9.0"]
+build-backend = "uv_build"
```

Comment:
I added this based on https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#declaring-the-build-backend

Click on the `uv-build` code block.